scipy/optimize/_differentialevolution.py

```diff
@@ -125,7 +125,7 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
         denoted by CR. Increasing this value allows a larger number of mutants
         to progress into the next generation, but at the risk of population
         stability.
-    rng : `numpy.random.Generator`, optional
+    rng : {None, int, `numpy.random.Generator`}, optional
```

Comment:
```suggestion
    rng : `numpy.random.Generator`, optional
```
Per the SPEC: "Also note the suggested language for the rng parameter docstring, which encourages the user to pass a Generator or None, but allows for other types accepted by numpy.random.default_rng (captured by the type annotation)." (Typing will come later.)